extensions/users-permissions/models/user.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const axios = require('axios');
+
+const subscribeToMailingList = async (values) => {
+  const { data } = await axios.post(`https://barmga-lambda.nyaladev.com/email-subscribe`, values);
+  return data;
+};
+
+module.exports = {
+  lifecycles: {
+    afterCreate: async (model, response, options) => {
+      if (model.emailSubscription && model.language) {
```

Comment:
please put all of this in a try/catch to make sure it fails silently if it fails .. and in the catch block use `strapi.log.error` instead of console.log